rbd-target-api.py

```diff
@@ -948,6 +950,17 @@ def clientlun(client_iqn):
         # this is a delete request
         if disk in lun_list:
             lun_list.remove(disk)
+
+            pool_name, image_name = disk.split('.')
+            with rados.Rados(conffile=settings.config.cephconf) as cluster:
+                with cluster.open_ioctx(pool_name) as ioctx:
+                    with rbd.Image(ioctx, image_name) as image:
+
+                        if len(image.list_lockers()):
```

Comment:
Nit: no need, just invoke the line below but wrap it w/ a ```try/except``` clause to catch ```rbd.ImageNotFound``` and ```rbd.InvalidArgument```.